infrastructure/environments/dplplat01/sites.yaml

```diff
@@ -50,3 +50,9 @@ sites:
     description: "The library site for Faxe"
     deploy_key: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIBkdxUoBx0ZAXfMfA0rRUNo2EcUK39fp0M/zKJPOcYx2"
     << : *default-release-image-source
+  customizable-canary:
```

Comment:
What is the differnce between "customizable" and "webmaster". If there is no difference between the two terms I think we should use only one of them.

Reply:
Yes - the term used for the `plan` attribute is `webmaster`. The name of the site (`customizable-canary`) is just there because that's what I created it as. When we're happy with the staging site feature I plan on deleting it again (so, very soon) so I didn't think it was super significant.

We really want a real customer on the plan to run a pilot site if possible so we can get their input